### PR TITLE
Add test for client installation with empty keytab file

### PR DESCRIPTION
**Pull request by netoarmando**
Missing test case for cf1301fb064fc230c780c4bc5eeccb723899f7b6.

https://pagure.io/freeipa/issue/7625

Signed-off-by: Armando Neto <abiagion@redhat.com>

Related Pull requests: #2308 and #2330.

### CODE REVIEW

File: ipatests/test_integration/test_forced_client_reenrollment.py [hunk 1]
```diff
@@ -160,6 +160,19 @@ def test_try_to_reenroll_with_incorrect_keytab(self, client):
...
 
+    def test_try_to_reenroll_with_empty_keytab(self, client):
+        """
+        Client re-enrollment with invalid (empty) client keytab file
+        """
+        self.restore_client()
+        self.check_client_host_entry()
+        try:
+            os.remove(CLIENT_KEYTAB)
+        except OSError:
```
Comment (tiran) [inline ipatests/test_integration/test_forced_client_reenrollment.py]: FYI, once we have Python 3 only, we can use ``FileNotFoundError`` here.

**Comment by tiran** [comment]
master:

* c049992c6f1a21ec084d6371c8ea287a53e194c0 Add test for client installation with empty keytab file
